source/common/upstream/upstream_impl.cc

```diff
@@ -2518,6 +2518,10 @@ bool BaseDynamicClusterImpl::updateDynamicHostList(
 
 Network::DnsLookupFamily
 getDnsLookupFamilyFromCluster(const envoy::config::cluster::v3::Cluster& cluster) {
```

Comment:
This function isnt really used and I really think its easier if we only use getDnsLookupFamilyFromEnum instead